src/parser/config/nodeTypesConfig.json

```diff
@@ -1948,10 +1948,10 @@
     },
     {
         "expression": "field_declaration_list",
-        "metrics": ["classes"],
```

Comment:
Was this a wrong mapping? If yes, would you please add a regression test? At least one test should have failed in my opinion.

Reply:
If I remember correctly, the issue was that this did work for C++, but not for Rust. As `class_specifier` and `struct_specifier` should be the more straight forward choice for C++ anyway, we decided to count them instead for C++.

Reply:
I do not know about Go, as there were no test cases provided by you for the classes metric for that language while mentioning the reason that there are no classes in Go. But I see now that we actually counted this as class for Go, mh. We might look into that again.